src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -97,6 +125,14 @@ export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit,
     return this._value;
   }
 
+/**
+ * Specifies an index for all the radio buttons in the group. If the index is not defined,
+ * the indices for individual radio buttons are set to their positions on load.
+ * This property supports
+ * [accessibility](https://developer.blackbaud.com/skyux/components/radio#accessibility)
```

Comment:
This link is busted. I think we can just remove the link.

Reply:
✔️